src/app/hooks/useGlean.ts

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useEffect } from "react";
+import Glean from "@mozilla/glean/web";
+import * as appEvents from "../../telemetry/generated/appEvents";
+
+// Custom hook that initializes Glean and returns the Glean objects required
+// to record data.
+export const useGlean = () => {
+  // Initialize Glean only on the first render
+  // of our custom hook.
+  useEffect(() => {
+    Glean.initialize("monitor-testing", true, {
+      // This will submit an events ping every time an event is recorded.
+      maxEvents: 1,
+    });
+
+    // Enable logging pings to the browser console.
+    Glean.setLogPings(true);
+
+    // Set the tag and sends pings to the debug ping viewer.
+    Glean.setDebugViewTag("monitor-testing");
```

Comment:
We might only want to enable this if `APP_ENV` has a particular value (e.g. `local`, `development` or `heroku` - @rhelmer probably knows the relevant values).

Reply:
`APP_ENV` can be accessed by front-end code with `NEXT_PUBLIC_APP_ENV`, the expected values are listed here: https://github.com/mozilla/blurts-server/blob/d1bf222bb285cfdb7cc6489db876997b24b89f20/.env-dist#L4-L6